### ISSUE TITLE
Support configuration of order

### ISSUE THREAD

**Issue by acka47** [skohub-io/skohub-vocabs#56]
...at best per level of hierarchy. Current example is https://test.skohub.io/dini-ag-kim/hochschulfaechersystematik/w3id.org/class/hochschulfaechersystematik/scheme.html

Currently, all levels are obviously ordered alphabetically by `prefLabel`. The original classification is ordered by notation on the first and second level and alphabetically on the third and lowest hierarchy level, see https://www.destatis.de/DE/Methoden/Klassifikationen/Bildung/studenten-pruefungsstatistik.pdf?__blob=publicationFile

To reach the same ordering in SkoHub, it would be great to somehow configure the ordering per level.

**Comment by acka47** [comment]
@TobiasNx thankfully made me aware that SkoHub vocabs out of the box supports configuring order as the HTML just follows the order of `skos:narrower` statements in the actual turtle file.

I've confirmed it for the Hochschulfächersystematik by adjusting the `skos:narrower` order, see https://github.com/dini-ag-kim/hochschulfaechersystematik/pull/16. So we can close this one as it has been resolved from the beginning.
